v2/cmd/asoctl/internal/importing/resource_importer_report.go

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package importing
+
+import (
+	"sync"
+
+	"github.com/go-logr/logr"
+	"golang.org/x/exp/maps"
+	"golang.org/x/exp/slices"
+)
+
+// resourceImportReport is used to generate a report of the resources that were imported (or not).
+type resourceImportReport struct {
+	content map[resourceImportReportKey]int
```

Comment:
Leave a comment explaining what exactly the int is/represents?

Reply:
Updated.